### PR TITLE
EDX-3755 Update model for XRT v2.1

### PR DESCRIPTION
**Pull request by weichou1229**
- add AddDiscoveredDeviceRequest for adding device without profile
- add ScanDeviceRequest for discovering the profile

### CODE REVIEW

**Review by cloudxxx8 — APPROVED** [review]
LGTM

File: xrtmodels/request.go [hunk 1]
```diff
@@ -60,6 +61,22 @@ type AddDeviceRequest struct {
...
 }
 
+type AddDiscoveredDeviceRequest struct {
+	BaseRequest `json:",inline"`
+	DeviceName  string               `json:"device"`
+	DeviceInfo  DiscoveredDeviceInfo `json:"device_info"`
+}
+
+type DiscoveredDeviceInfo struct {
+	Protocols map[string]map[string]interface{} `json:"protocols"`
```
Comment (cloudxxx8) [inline xrtmodels/request.go:71]: use `any` instead of `interface{}`

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: xrtmodels/request.go (hunk 1 above)
Comment (weichou1229) [reply xrtmodels/request.go]: Updated.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**